deployment/docker-compose.yml

```diff
@@ -35,7 +35,7 @@ db:
   container_name: mezzanine-db
   image: kartoza/postgis:9.4-2.1
   volumes:
-    - ./pg/postgres_data:/var/lib/postgresql
```

Comment:
I would like to leave it in 'persist' mode for now until we get rancher running nicely and we can talk the db storage container for preservation when other containers are deleted.